src/pages/en/guides/markdown-content.md

```diff
@@ -368,22 +375,120 @@ export default {
+- **Prism:** Choose from the available [Prism Themes](https://github.com/PrismJS/prism-themes) and see the [list of languages supported by Prism](https://prismjs.com/#supported-languages) for options and usage.
+- **Built-in components:** See the [`<Code />` Astro component](/en/reference/builtin-components/#code-) powered by Shiki, and the [`<Prism />` Astro component](/en/reference/builtin-components/#prism-) for a Prism-based version.
+
+#### Create a custom Shiki theme
+
+> **Intended audience:** This is for users that need full control over the colors and font styles in their syntax highlighting. We understand that some users have custom Prism stylesheets and may want to try Shiki. Before reading on, note that Shiki themes **are** more complex and granular compared to Prism!
+
+Shiki themes use the TextMate / VS Code theme format. This allows Shiki to directly port popular editor themes into your syntax highlighting. This is a departure from the stylesheet-based approach that Prism uses, unless you opt for [the `css-variables` theme option](#option-2-use-the-css-variable-theme).
+
+##### Option 1: create a custom JSON object
+
+Themes are created using an array of "tokens," built to target selectors in your compiled code blocks. These are passed via the `theme.settings` key on your `@astrojs/markdown-remark` configuration:
+
+```js
+// astro.config.mjs
+export default {
+  markdownOptions: {
+    render: [
+      '@astrojs/markdown-remark',
+      {
+        shikiConfig: {
+          theme: {
+            name: 'my-custom-theme',
+            // can be 'light' or 'dark'
+            type: 'dark',
+            // list of tokens here
+            settings: [{
+                // required object: sets base foreground and background
+                settings: {
+                  foreground: '#fff',
+                  background: '#000',
+                },
+              },
+              {
+                // ex: target all emphasized elements
+                scope: 'emphasis',
+                settings: {
+                  fontStyle: 'italic',
+                },
+              },
+            ...
+            ],
+          },
+        },
+      },
+    ],
+  },
+};
+```
+
+We suggest [using an existing theme](https://github.com/shikijs/shiki/blob/main/scripts/themeSources.ts#L40-L68) as a base before getting started. This way, you can find-and-replace with your desired styling before diving _too_ deeply into all available options (spoiler: there's quite a few!)
+
+You can also [check the custom theme for this very website](https://github.com/withastro/docs/blob/main/syntax-highlighting-theme.ts) (docs.astro.build) to see how we set everything up.
+
+##### Option 2: use the `css-variables` theme
+
+If maintaining a large JSON object feels daunting, you can always try Shiki's `css-variables` theme. Enable this using the `theme` configuration option:
+
+```js
+// astro.config.mjs
```

Comment:
Just pointing out that if we decide to go through and change `.mjs` to `.js`, then this gets updated. (More reason to KEEP using `.mjs` for now, I think!)